REUSE.toml

```diff
@@ -3,6 +3,12 @@
 
 version = 1
 
+[[exclude]]
+path = [
+    "inventory/docs/inventory-er-diagram.svg",
+    "inventory/docs/output.dbml"
```

Comment:
It's better to put these in the `annotations` section below, instead of excluding them.